spec/unit/lib/cloud_controller/diego/lifecycle_data_spec.rb

```diff
@@ -1,5 +1,5 @@
 require 'spec_helper'
```

Comment:
Shouldn't that file be located at `spec/unit/lib/cloud_controller/diego/buildpack/lifecycle_data_spec.rb`

Reply:
good point, the file has been moved